controllers/user.go

```diff
@@ -563,10 +573,18 @@ func (c *ApiController) SetPassword() {
 	targetUser.NeedUpdatePassword = false
 	targetUser.LastChangePasswordTime = util.GetCurrentTime()
 
-	_, err = object.UpdateUser(userId, targetUser, []string{"password", "need_update_password", "password_type", "last_change_password_time"}, false)
-	if err != nil {
-		c.ResponseError(err.Error())
-		return
+	if user.Ldap == "" {
```

Comment:
how about user want login with password store in casdoor? maybe we can create a extra button named `Resert LDAP password`

Reply:
The password authentication source for LDAP users can only come from the LDAP server. Even if a password is configured for the user in the database, the login verification will still be associated with LDAP. Therefore, the local password is redundant for the user, and since users do not know whether they are LDAP users, this may cause confusion.

Reply:
ok